__tests__/app.test.js

```diff
@@ -482,3 +483,28 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: respond with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users).toBeInstanceOf(Array);
+      });
+  });
+
+  it("200: each user has 3 properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+        expect(body.users[0]).toMatchObject({
```

Comment:
would be better to iterate through and check each user has these properties than just check the first user.